javascripts/mindmap.js

```diff
@@ -381,15 +381,15 @@ function update(source) {
     .attr('id', d => `node--${d.id}`)
     .attr("transform", function (d) {
       return "translate(" + source.y0 + "," + source.x0 + ")";
-    })
-    .on('click', click);
+    });
 
   nodeEnter.append("circle")
     .attr('class', 'nodeCircle')
     .attr("r", 0)
     .style("fill", function (d) {
       return d._children ? "lightsteelblue" : "#fff";
-    });
+    })
+    .on('click', click);
```

Comment:
What's this?

Reply:
So before, the click was on the circle, and the text, aka the node, now it's only the circle, ideally I would love to slip the set focus, update breadcrumbs, and the open/close click actions, but that's beyond me, so I just did this patch to start with.